src/main/resources/db/migration/V002__create_users_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE SEQUENCE users_seq START WITH 1 INCREMENT BY 1;
+
+CREATE TABLE USERS(
```

Comment:
```suggestion
CREATE TABLE "USER"(
```
Nome de tabela é bom usar singular. Coloquei entre aspas pq USER é palavra reservada no PostgreSQL, daí com aspas ele entende que é o nome da tabela mesmo

Reply:
Alterado